website/code_issues.qmd

```diff
@@ -5,11 +5,45 @@ format:
     css: style.css
 ---
 
-# Issue 1
+# `T`/`F` Instead of `TRUE`/`FALSE` 
 
 ## Problem
+You are writing the abbreviated forms of `TRUE` and `FALSE`, `T` and `F` or you are using `T` or `F` as names.
 
 ## Solution
+Change the abbreviation to the full words or use a different variable name. 
+
+## Details
```

Comment:
In the other issues the Details section is header 3 - do we want to keep the same headers for EVERY issue? I think its totally its fine to deviate but I think the little dropdown on the upper right-hand will look different.

Reply:
Oh, yeah! I thought I put it as header 2 there as well but you are right.